core/streamer.py

```
@@ -569,21 +569,31 @@ def _start_proxy_health_monitor(self):
+                            "[FAIL_CLOSED] proxy /health fail-closed 지속으로"
+                            f" 측정 중단함 (subject {self.subject_index})"
+                        )
+                        try:
+                            self.close_session()
+                        except Exception as e:
+                            logger.warning(f"close_session 실패 (무시): {e}")
+                        finally:
+                            # close 가 raise 하면 break 를 못 타 루프가 헛돎
+                            try:
+                                self.close()
+                            except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)